examples/ucp_client_server.c

```diff
@@ -51,6 +53,12 @@ static uint16_t server_port    = DEFAULT_PORT;
 static int num_iterations      = DEFAULT_NUM_ITERATIONS;
 
 
+typedef enum {
+    DATATYPE_IOV    = ucp_dt_make_iov(),
+    DATATYPE_CONTIG = ucp_dt_make_contig(1)
+} datatype_t;
```

Comment:
Maybe we don't need defining this, we can use UCP_DATATYPE_CONTIG and UCP_DATATYPE_IOV directly, and only calculate ucp_dt_make_iov() and ucp_dt_make_contig(1) at the place really need it.

Reply:
Done